src/main/java/racingcar/Application.java

```diff
@@ -1,7 +1,120 @@
 package racingcar;
 
+import camp.nextstep.edu.missionutils.Console;
+
+import java.util.*;
+
 public class Application {
     public static void main(String[] args) {
-        // TODO 구현 진행
+        List<String> carNames = receiveCarNames();
+
+        List<Car> cars = createCars(carNames);
+
+        int tryCount = receiveTryCount();
+
+        runRacingGame(cars, tryCount);
+
+        printWinners(findWinners(cars));
+    }
+
+
+    private static List<String> receiveCarNames() {
+        List<String> carNames;
+        while (true) {
+            System.out.println("경주할 자동차 이름을 입력하세요. (이름은 쉼표(,) 기준으로 구분)");
+            String input = Console.readLine();
+            carNames = Arrays.asList(input.split(",", -1));
+            try { // 예외가 발생한다면? -> catch
+                validateCarNames(carNames); // 유효성 검사
+                break;
+            }
+            catch (IllegalArgumentException e) {
+                System.out.println("[ERROR] 올바른 형식의 자동차 이름을 입력하세요.");
+            }
+        }
+        return carNames;
+    }
+
+    private static void validateCarNames(List<String> carNames) {
+        for (String inputName : carNames) {
+            if (inputName.isEmpty() || inputName.length() > 5) {
```

Comment:
```suggestion
            if (inputName.isEmpty() || inputName.length() > MAX_NAME_LENGTH) {
```
이렇게 클래스에 상수를 따로 선언해서 수정했을 때 어떤 이점이 있을까요?

Reply:
코드가 길고 복잡해졌을 때 숫자 5가 무엇을 나타내는지 혼동이 올 수 있습니다. 
이렇게 따로 상수를 선언했을 경우 보다 명확하고 이해하기 쉬워집니다.
또한 추후 값을 변경해야할 경우에 5를 각각 수정하는 것보다 선언해둔 상수를 수정하는 것이 편리하기 때문에 유지보수가 용이하다는 이점이 있습니다.